README.md

```diff
@@ -24,10 +39,17 @@ composer require rapidez/blade-components
 And make sure these colors are present in your Tailwind config:
 ```js
 colors: {
-    neutral: '#334155',
-    inactive: '#64748b',
-    border: '#e7ebef',
-    disabled: '#ebe8de',
+    colors: {
+        primary: {
+            DEFAULT: '#2FBC85',
+            text: '#FFFFFF',
+        },
+        neutral: '#334155',
+        inactive: '#64748B',
+        border: '#E7EBEF',
+        disabled: '#EBE8DE',
+        enhanced: '#36B422'
```

Comment:
Looking forward to the final version of this.

Reply:
This will be a follow up based on the feedback here: https://github.com/rapidez/core/pull/580. I will create a new PR so we can see the difference between both PR's